h/accounts/models.py

```diff
@@ -67,6 +67,21 @@ class User(Base):
                           nullable=False,
                           unique=True)
 
+    # The display name which will be used when rendering an annotation.
```

Comment:
Sorry to be such a pain. The colon after the comment character (`#:`) in the equivalent comments in `annotation.py` are important, because they allow the attribute to be [documented by Sphinx automatically](http://www.sphinx-doc.org/en/stable/ext/autodoc.html#directive-autoattribute). We're not generating documentation for these modules at the moment, but I'd like doc comments like this to be visible when we do.